classes/announcement/PKPAnnouncementDAO.inc.php

```diff
@@ -338,7 +338,7 @@ function &getAnnouncementsNotExpiredByAssocId($assocType, $assocId, $rangeInfo =
 			FROM announcements
 			WHERE assoc_type = ?
 				AND assoc_id = ?
-				AND (date_expire IS NULL OR date_expire > CURRENT_DATE)
+				AND (date_expire IS NULL OR date_expire >= CURRENT_DATE)
 				AND (date_posted < CURRENT_DATE)
```

Comment:
And this is my fault too for skimming the code so quickly. @gpieri, can you submit a PR that corrects the earlier one? I'll port it around.